.github/workflows/pimcore-skeleton.yml

```
@@ -0,0 +1,61 @@
+name: Test Pimcore Skeleton
+
+on:
+  # Trigger the workflow on push or pull request, but only for the 10.x branch
+  # See: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration
+  push:
+    branches:
+    - 10.x
```

let's use a regex here so that we don't have to change the config with minor release.

```suggestion
    - "[0-9]+.x"
```